deploy/crds/crd-certificaterequests.yaml

```diff
@@ -8,8 +8,7 @@ metadata:
     app: '{{ template "cert-manager.name" . }}'
     app.kubernetes.io/name: '{{ template "cert-manager.name" . }}'
     app.kubernetes.io/instance: '{{ .Release.Name }}'
-    app.kubernetes.io/managed-by: '{{ .Release.Service }}'
-    helm.sh/chart: '{{ template "cert-manager.chart" . }}'
+    # {{- include "labels" . | nindent 4 }}
```

Comment:
This seems to leave an empty comment in the generated YAML:

```yaml
# Source: cert-manager/templates/templates.out
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  name: certificaterequests.cert-manager.io
  annotations:
    cert-manager.io/inject-ca-from-secret: 'cert-manager/cert-manager-webhook-ca'
  labels:
    app: 'cert-manager'
    app.kubernetes.io/name: 'cert-manager'
    app.kubernetes.io/instance: 'cert-manager'
    #
    app.kubernetes.io/version: "v1.4.0-145-043bbd283eb6dd-dirty"
    app.kubernetes.io/managed-by: Helm
    helm.sh/chart: cert-manager-v1.4.0-145-043bbd283eb6dd-dirty
```

Any way to have it not do that? Or perhaps make it a meaningful comment like `# Generated labels`

Reply:
`# Generated labels` is probably the best solution. The CRDs have to be valid YAML, so all these Helm template structures has to be part of a comment.